FATN_to_USB/fatn_to_usb/main.py

```
@@ -0,0 +1,357 @@
+
+        FATN_parse = GetFATNUrl()
+        FATN = urlopen(url)
+        FATN_parse.feed(str(FATN.read()))
+
+        if not FATN_parse.FATNurl:
+            print('❌ No Dropbox link found on FATN website')
+            return None
+
+        # Get actual Dropbox URL from tinyurl
+        print(f'Resolving redirect: {FATN_parse.FATNurl}')
+        FATNdropboxurl = urlopen(FATN_parse.FATNurl)
```

Ruff (0.14.8)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)